samples: samples/shields/x_nucleo_iks01a2/standard : fails on NXP platforms

### Describe the bug

- frdm_k64f
- lpcxpresso55s28
- mimxrt1060_evk@C/mimxrt1062/hyperflash
- 

c683b044d9884abbdb8c42843c446dad81fe0812 is the first bad commit
```
commit c683b044d9884abbdb8c42843c446dad81fe0812
Author: Benjamin Cabé <benjamin@zephyrproject.org>
Date:   Tue Sep 2 12:16:59 2025 +0200

    shields: devicetree: use arduino-header-r3.h constants in all shields
    
    Make shield overlays less error prone and easier to maintain by
    switching to GPIO pin constants defined in arduino-header-r3.h.
    
    Signed-off-by: Benjamin Cabé <benjamin@zephyrproject.org>
```

### Regression

- [ ] This is a regression.

### Steps to reproduce

scripts/twister -p frdm_k64f -j 12 -A ../zsdk/boards -T ../zsdk -T samples/shields/x_nucleo_iks01a2/standard

### Relevant log output

```shell
-- Found host-tools: zephyr 0.17.1 (/opt/toolchains/zephyr-sdk-0.17.1-rc4)-- Found toolchain: zephyr 0.17.1 (/opt/toolchains/zephyr-sdk-0.17.1-rc4)-- Found Dtc: /opt/toolchains/zephyr-sdk-0.17.1-rc4/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.7.0", minimum required is "1.4.6") -- Found BOARD.dts: /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/boards/nxp/frdm_k64f/frdm_k64f.dts-- Found devicetree overlay: /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/boards/shields/x_nucleo_iks01a2/x_nucleo_iks01a2.overlaydevicetree error: /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/boards/shields/x_nucleo_iks01a2/x_nucleo_iks01a2.overlay:34 (column 2): parse error: expected number or parenthesized expressionCMake Error at /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/cmake/modules/dts.cmake:306 (execute_process): execute_process failed command indexes: 1: "Child return code: 1"Call Stack (most recent call first): /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/cmake/modules/zephyr_default.cmake:131 (include) /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include) /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:92 (include_boilerplate) CMakeLists.txt:8 (find_package)-- Configuring incomplete, errors occurred!
```

### Impact

Annoyance – Minor irritation; no significant impact on usability or functionality.

### Environment

- OS: ubuntu
- tool-chain: 0.17.1
- version: 8f85d8e62ac5bec45e703086ee28583e9f67baac

### Additional Context

_No response_

--- a/boards/shields/waveshare_epaper/waveshare_epaper_gdeh0213b1.overlay
+++ b/boards/shields/waveshare_epaper/waveshare_epaper_gdeh0213b1.overlay
@@ -16,7 +16,7 @@
 		compatible = "zephyr,mipi-dbi-spi";
 		spi-dev = <&arduino_spi>;
 		dc-gpios = <&arduino_header ARDUINO_HEADER_R3_D9 GPIO_ACTIVE_HIGH>;
-		reset-gpios = <&arduino_header ARDUINO_HEADER_R3_D8 GPIO_ACTIVE_LOW
+		reset-gpios = <&arduino_header ARDUINO_HEADER_R3_D8 GPIO_ACTIVE_LOW>;
 		#address-cells = <1>;
 		#size-cells = <0>;
 

--- a/boards/shields/waveshare_epaper/waveshare_epaper_gdew042t2-p.overlay
+++ b/boards/shields/waveshare_epaper/waveshare_epaper_gdew042t2-p.overlay
@@ -16,7 +16,7 @@
 		compatible = "zephyr,mipi-dbi-spi";
 		spi-dev = <&arduino_spi>;
 		dc-gpios = <&arduino_header ARDUINO_HEADER_R3_D9 GPIO_ACTIVE_HIGH>;
-		reset-gpios = <&arduino_header ARDUINO_HEADER_R3_D8 GPIO_ACTIVE_LOW
+		reset-gpios = <&arduino_header ARDUINO_HEADER_R3_D8 GPIO_ACTIVE_LOW>;
 		write-only;
 		#address-cells = <1>;
 		#size-cells = <0>;

--- a/boards/shields/x_nucleo_iks01a2/x_nucleo_iks01a2.overlay
+++ b/boards/shields/x_nucleo_iks01a2/x_nucleo_iks01a2.overlay
@@ -30,7 +30,7 @@
 	lsm6dsl_6b_x_nucleo_iks01a2: lsm6dsl@6b {
 		compatible = "st,lsm6dsl";
 		reg = <0x6b>;
-		irq-gpios = <&arduino_header ARDUINO_HEADER_R3_D4 GPIO_ACTIVE_HIGH
+		irq-gpios = <&arduino_header ARDUINO_HEADER_R3_D4 GPIO_ACTIVE_HIGH>;
 	};
 
 	lsm303agr_magn_1e_x_nucleo_iks01a2: lsm303agr-magn@1e {

